GDJS/Runtime/scenestack.ts

```diff
@@ -126,7 +126,8 @@ namespace gdjs {
      */
     push(
```

Comment:
Important: can you rework this to align with: https://github.com/4ian/GDevelop/blob/e1835d1144ae25667c6e01d61c717937382649b0/GDJS/Runtime/scenestack.ts#L140-L155

See also _loadNewScene: https://github.com/4ian/GDevelop/blob/e1835d1144ae25667c6e01d61c717937382649b0/GDJS/Runtime/scenestack.ts#L186

Basically: we switch to an object for parameters, rather than these ugly list of "newSceneName, undefined, options". We check the type to know if it's the deprecated arguments or the new ones.